### PR TITLE
net-mgmt/telegraf: add unbound

### PR DESCRIPTION
**Pull request by mimugmail**
Add unbound statistics and change permissions (add group wheel)

### CODE REVIEW

File: net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/General.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
             <default>0</default>
             <Required>Y</Required>
         </enabled>
+        <wheelgroup type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/General.xml:12]: model bump please :D

File: net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf [hunk 2]
```diff
@@ -263,4 +263,11 @@
   json_string_fields = ["*"]
 {% endif %}
 
+{% if helpers.exists('OPNsense.telegraf.input.intrusion_detection_alerts') and OPNsense.telegraf.input.intrusion_detection_alerts == '1' %}
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf:266]: This doesn't look right at all

Comment (mimugmail) [reply net-mgmt/telegraf/src/opnsense/service/templates/OPNsense/Telegraf/telegraf.conf]: Indeed .. always good to have a review

File: net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <model>
     <mount>//OPNsense/telegraf/input</mount>
     <description>Telegraf inputs configuration</description>
-    <version>1.0.3</version>
+    <version>1.0.4</version>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml:4]:
```suggestion
    <version>1.0.3</version>
```

File: net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml [hunk 4]
```diff
@@ -93,5 +93,9 @@
             <default>0</default>
             <Required>N</Required>
         </intrusion_detection_alerts>
+        <unbound type="BooleanField">
+            <default>0</default>
```
Comment (fichtner) [inline net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml:97]: I don't want to set the practice to inject default values to non-required (empty) vars.

```suggestion
```

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**